spec/features/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page', type: :feature do
+  describe 'show page' do
+    before :each do
+      # @user = User.create(name: 'Tom', bio: 'Teacher from Mexico.', photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+      # @id = @user.id
+      # Post.create(user_id: @user, title: 'Hello', text: 'Make a new comment on this post')
+      # Post.create(user_id: @user, title: 'hello', text: 'Make a new comment on this post')
+      visit '/users/83'
+    end
+
```

Comment:
- [ ] @team as requested by the previous reviewer kindly add tests to check the number of posts the user has and also the test that you can see the profile picture.
- [ ] [optional] it's not good practice to have commented code in your codebase kindly remove it.
-

Reply:
Hello there! in my computer the test renders well.  I don't know what is the problem, I'm very stressed with this testing.

Reply:
i have less than two days to complete the rest of the project, please if you can help me i'm really appreciate that.